submissions/vakulinina/js-dom/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>DOM</title>
+</head>
+
+<body>
+  <header>
+    <button class="menu-button"><img src="img/menu-icon.svg" alt="menu"></button>
+    <h1>Coffee drinks</h1>
+  </header>
+  <main>
+    <nav class="nav-bar">
+      <ul class="nav-buttons">
+        <li><button class="nav-button" id="americano">Americano</button></li>
```

Comment:
You can generate this list from your script file

Reply:
May I ask what for? Will it have an impact on performance or something else?

Reply:
It will help you depend only on your `coffeeDrinks` list. If you'll change some element in your hardcoded navigation items , your app can be broken.